onnxruntime/core/providers/dml/DmlExecutionProvider/src/DmlCommon.h

```diff
@@ -6,6 +6,11 @@
 #include <assert.h>
 #include "core/providers/dml/OperatorAuthorHelper/Common.h"
 
+template <class T>
+auto unmove_ptr(T&& t) {
+  return &static_cast<T&>(t);
```

Comment:
[nit] Consistent indent with rest of file (no 🔎 needed to efficiently read the blocks of code :b).